src/main/java/com/google/step/youtube/DescriptionParser.java

```diff
@@ -0,0 +1,14 @@
+package com.google.step.youtube;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.regex.Matcher;
+
+/**
+ * Class for parsing the descriptions of YouTube videos for promotional codes and affiliate links. 
+ */
+public class DescriptionParser {
+
```

Comment:
nit: Add a TODO comment saying the implementation is coming and this is just a placeholder class.  

nit: Also, I noticed you didn't reference the Parser/Scraper.  Typically you don't include methods/classes, etc unless you're actually using them in the CL that you're checking in.  What often happens is that subsequent CL that references the new methods/classes never gets checked in and then there's dead code.

Reply:
Added another level of abstraction shown in updated diagram.